BonusCalcListener/Gateway/TimesheetGateway.cs

```diff
@@ -23,8 +23,10 @@ public async Task<Timesheet> GetCurrentTimeSheetForOperative(string operativeId,
         {
             if (!closedTime.HasValue) throw new ArgumentNullException(nameof(closedTime));
 
+            var weekStart = WeekStartForClosedTime(closedTime);
```

Comment:
@Asura5 I'm making an assumption here that the timestamp coming from RepairsHub is in UTC and that we need to convert it to local time and then calculate the week start from that. If it's not the case then I need to rework this.

Reply:
There's not explicit conversion from the value supplied in the frontend into UTC - so I believe it's stored as local time. @NeilvB is the completion time sent from the frontend exactly as entered?

Reply:
Looking at the code it seems they call `toIsoString` on the date object which [according to MDN][1] is always UTC.

[1]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString

Reply:
Just to add - we send up completion payloads from two places. I've checked and tested both and we construct a UTC date in each.